alpha/config/kConfCacheManager.php

```diff
@@ -127,7 +127,7 @@ public static function load ($mapName, $key=null)
 				$key = self::loadKey();
 
 			$map = $cacheObj->load($key, $mapName);
-			if($map)
+			if(!empty($map))
```

Comment:
The ->load return mergeMaps content, and if no content , it should return empty array , no need to check !empty